<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="DESKTOP-O53FF8H" failures="0" tests="25" name="org.botShreyasi.passiveRefferals.AppTest" time="136.387" errors="1" timestamp="2023-12-20T10:54:30 IST" skipped="0">
  <testcase classname="org.botShreyasi.passiveRefferals.AppTest" name="at" time="31.054"/>
  <system-out/>
  <testcase classname="org.botShreyasi.passiveRefferals.AppTest" name="at" time="0.007"/>
  <system-out/>
  <testcase classname="org.botShreyasi.passiveRefferals.AppTest" name="at" time="0.016"/>
  <system-out/>
  <testcase classname="org.botShreyasi.passiveRefferals.AppTest" name="at" time="16.093"/>
  <system-out/>
  <testcase classname="org.botShreyasi.passiveRefferals.AppTest" name="at" time="0.022"/>
  <system-out/>
  <testcase classname="org.botShreyasi.passiveRefferals.AppTest" name="at" time="0.084"/>
  <system-out/>
  <testcase classname="org.botShreyasi.passiveRefferals.AppTest" name="at" time="0.037"/>
  <system-out/>
  <testcase classname="org.botShreyasi.passiveRefferals.AppTest" name="at" time="12.673"/>
  <system-out/>
  <testcase classname="org.botShreyasi.passiveRefferals.AppTest" name="at" time="21.960"/>
  <system-out/>
  <testcase classname="org.botShreyasi.passiveRefferals.AppTest" name="at" time="0.083"/>
  <system-out/>
  <testcase classname="org.botShreyasi.passiveRefferals.AppTest" name="at" time="0.046"/>
  <system-out/>
  <testcase classname="org.botShreyasi.passiveRefferals.AppTest" name="at" time="0.130"/>
  <system-out/>
  <testcase classname="org.botShreyasi.passiveRefferals.AppTest" name="at" time="0.125"/>
  <system-out/>
  <testcase classname="org.botShreyasi.passiveRefferals.AppTest" name="at" time="0.290"/>
  <system-out/>
  <testcase classname="org.botShreyasi.passiveRefferals.AppTest" name="at" time="28.375"/>
  <system-out/>
  <testcase classname="org.botShreyasi.passiveRefferals.AppTest" name="at" time="0.067"/>
  <system-out/>
  <testcase classname="org.botShreyasi.passiveRefferals.AppTest" name="at" time="0.025"/>
  <system-out/>
  <testcase classname="org.botShreyasi.passiveRefferals.AppTest" name="at" time="0.035"/>
  <system-out/>
  <testcase classname="org.botShreyasi.passiveRefferals.AppTest" name="tearDown" time="0.144">
    <error message="JdkWebSocket initial request execution error
Build info: version: &#039;4.14.0&#039;, revision: &#039;e5a96349ce&#039;
System info: os.name: &#039;Windows 10&#039;, os.arch: &#039;amd64&#039;, os.version: &#039;10.0&#039;, java.version: &#039;17.0.8.1&#039;
Driver info: driver.version: FirefoxDriver" type="org.openqa.selenium.remote.http.ConnectionFailedException">
      <![CDATA[org.openqa.selenium.remote.http.ConnectionFailedException: JdkWebSocket initial request execution error
Build info: version: '4.14.0', revision: 'e5a96349ce'
System info: os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '17.0.8.1'
Driver info: driver.version: FirefoxDriver
at org.openqa.selenium.remote.http.jdk.JdkHttpClient.openSocket(JdkHttpClient.java:243)
at org.openqa.selenium.devtools.Connection.<init>(Connection.java:83)
at org.openqa.selenium.firefox.FirefoxDriver.maybeGetDevTools(FirefoxDriver.java:334)
at org.openqa.selenium.remote.RemoteWebDriver.quit(RemoteWebDriver.java:430)
at org.openqa.selenium.firefox.FirefoxDriver.quit(FirefoxDriver.java:392)
at org.botShreyasi.passiveRefferals.AppTest.tearDown(AppTest.java:100)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:62)
at org.testng.internal.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:385)
at org.testng.internal.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:321)
at org.testng.internal.TestMethodWorker.invokeAfterClassMethods(TestMethodWorker.java:217)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:130)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.access$000(SuiteRunner.java:28)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:418)
at org.testng.internal.thread.ThreadUtil.lambda$execute$0(ThreadUtil.java:64)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
Caused by: java.net.ConnectException
at java.net.http/jdk.internal.net.http.common.Utils.toConnectException(Utils.java:1055)
at java.net.http/jdk.internal.net.http.PlainHttpConnection.connectAsync(PlainHttpConnection.java:198)
at java.net.http/jdk.internal.net.http.PlainHttpConnection.checkRetryConnect(PlainHttpConnection.java:230)
at java.net.http/jdk.internal.net.http.PlainHttpConnection.lambda$connectAsync$1(PlainHttpConnection.java:206)
at java.base/java.util.concurrent.CompletableFuture.uniHandle(CompletableFuture.java:934)
at java.base/java.util.concurrent.CompletableFuture$UniHandle.tryFire(CompletableFuture.java:911)
at java.base/java.util.concurrent.CompletableFuture.postComplete(CompletableFuture.java:510)
at java.base/java.util.concurrent.CompletableFuture$AsyncSupply.run(CompletableFuture.java:1773)
... 3 more
Caused by: java.nio.channels.ClosedChannelException
at java.base/sun.nio.ch.SocketChannelImpl.ensureOpen(SocketChannelImpl.java:195)
at java.base/sun.nio.ch.SocketChannelImpl.beginConnect(SocketChannelImpl.java:760)
at java.base/sun.nio.ch.SocketChannelImpl.connect(SocketChannelImpl.java:848)
at java.net.http/jdk.internal.net.http.PlainHttpConnection.lambda$connectAsync$0(PlainHttpConnection.java:183)
at java.base/java.security.AccessController.doPrivileged(AccessController.java:569)
at java.net.http/jdk.internal.net.http.PlainHttpConnection.connectAsync(PlainHttpConnection.java:185)
... 9 more
]]>
    </error>
  </testcase> <!-- tearDown -->
  <system-out/>
  <testcase classname="org.botShreyasi.passiveRefferals.AppTest" name="at" time="0.293"/>
  <system-out/>
  <testcase classname="org.botShreyasi.passiveRefferals.AppTest" name="at" time="24.122"/>
  <system-out/>
  <testcase classname="org.botShreyasi.passiveRefferals.AppTest" name="at" time="0.600"/>
  <system-out/>
  <testcase classname="org.botShreyasi.passiveRefferals.AppTest" name="at" time="0.058"/>
  <system-out/>
  <testcase classname="org.botShreyasi.passiveRefferals.AppTest" name="at" time="0.009"/>
  <system-out/>
  <testcase classname="org.botShreyasi.passiveRefferals.AppTest" name="at" time="0.039"/>
  <system-out/>
</testsuite> <!-- org.botShreyasi.passiveRefferals.AppTest -->
